docker/worker/testdata/expected_127.diff

```diff
@@ -0,0 +1,19 @@
+diff --git a/BLAH-127.yaml b/BLAH-127.yaml
```

Comment:
At some point, we should think of renaming these test files to be more readable, like instead of _127, it could be _no_regress_commit ? can be another cl later at some point.

Reply:
ack.